worker/backup_processor.go

```diff
@@ -310,3 +323,49 @@ func writeKVList(list *bpb.KVList, w io.Writer) error {
 	_, err = w.Write(buf)
 	return err
 }
+
+func checkAndGetDropOp(key []byte, l *posting.List, readTs uint64) (*pb.DropOperation, error) {
+	isDropOpKey, err := x.IsDropOpKey(key)
+	if err != nil || !isDropOpKey {
+		return nil, err
+	}
+
+	vals, err := l.AllValues(readTs)
+	if err != nil {
+		return nil, errors.Wrapf(err, "cannot read value of dgraph.drop.op")
+	}
+	switch len(vals) {
+	case 0:
+		// do nothing, it means this one was deleted with S * * deletion.
+		// So, no need to consider it.
+		return nil, nil
+	case 1:
+		val, ok := vals[0].Value.([]byte)
+		if !ok {
+			return nil, errors.Errorf("cannot convert value of dgraph.drop.op to byte array")
```

Comment:
Add the actual type of the interface to the error. `reflect.TypeOf(...)` would give you this information.

Reply:
Done